libcontainer/apparmor/apparmor.go

```diff
@@ -35,14 +33,13 @@ func setProcAttr(attr, value string) error {
 		return err
 	}
 
-	_, err = fmt.Fprintf(f, "%s", value)
+	_, err = fmt.Fprint(f, value)
```

Comment:
It's better to use
```go
_, err := `f.WriteString(value)`
```
instead I guess.

Reply:
Ah, yes, didn't think of that 👍, will update